ui/accordion.reel/accordion.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @module accordion
+ * @requires montage/ui/montage
+ * @requires montage/ui/component
+ */
+var Montage = require("montage").Montage,
+    Component = require("montage/ui/component").Component;
+
+/**
+ * @class Accordion
+ * @extends Component
+ */
+exports.Accordion = Montage.create(Component, {
```

Comment:
Should we use `Montage.create(Component, {})` or `Component.specialize({})`? Other components seem to use the later. Not sure what the best practice is.